tests/ci/config/config.go

```diff
@@ -37,6 +37,8 @@ type TestConfig struct {
 	ClusterDetailFile             string
 	ClusterInstallLogArtifactFile string
 	ClusterAdminFile              string
+	ClusterKubeconfigIdpFile      string
```

Comment:
In that file you are writing user/password information from idp, so better to rename it to something like `ClusterIDPUsernamePasswords`